vespa/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM vespaengine/vespa:7.104.46
```

Comment:
this is a different number to the other pr?

Reply:
this is the right one